3rdParty/fuerte/include/fuerte/message.h

```diff
@@ -193,8 +194,13 @@ class Request final : public Message {
   /// @brief request header
   RequestHeader header;
 
+
   MessageType type() const override { return MessageType::Request; }
   MessageHeader const& messageHeader() const override { return header; }
+  void setFuzzReqHeader(std::string& fuzzHeader) {_fuzzReqHeader = fuzzHeader;}
+  std::optional<std::string> getFuzzReqHeader() const {return _fuzzReqHeader;}
+  void setFuzzerReq(bool isFuzzerReq) {_isFuzzerReq = isFuzzerReq;}
+  bool getFuzzerReq() const {return _isFuzzerReq;}
```

Comment:
Code formatting (see lines above):
```suggestion
  void setFuzzReqHeader(std::string& fuzzHeader) { _fuzzReqHeader = fuzzHeader; }
  std::optional<std::string> getFuzzReqHeader() const { return _fuzzReqHeader; }
  void setFuzzerReq(bool isFuzzerReq) { _isFuzzerReq = isFuzzerReq; }
  bool getFuzzerReq() const { return _isFuzzerReq; }
```